packages/html-manager/src/libembed-amd.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
+const __widgets_cdn__ = (window as any).__widgets_cdn__ || 'https://unpkg.com';
```

Comment:
I think it might be a bit cleaner if we make things more specific, like `__jupyter_widgets_cdn__`, or if we could figure out a nice way to not have a global variable.

Reply:
Updated the var name to `__jupyter_widgets_cdn__` : )